app/assets/javascripts/xml_navigator.js

```diff
@@ -45,21 +45,21 @@
   }
   Navigator.prototype.prev = function() {
     this.index = this.index  - 1;
-    if(this.index == NaN || this.index < 0){this.index=0;}
+    if(this.index.isNaN || this.index < 0){this.index=0;}
```

Comment:
where is the isNaN property coming from?  there is a javascript method isNaN(value) that will return true or false but I don't think there is an isNaN property on NaN